vitest.config.js

```diff
@@ -7,5 +7,6 @@ export default defineConfig({
         singleFork: true,
       },
     },
+    testTimeout: 180_000,
```

Comment:
Seems like previously Vitest wasn't enforcing the timeout (which defaults to 5s). 🤔 AFAICT there's no actual difference in test time.